Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -166,7 +166,9 @@ file that was distributed with this source code.
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? label|trans({}, translation_domain) -}}
+            {%- if sonata_admin.options['use_icheck'] -%}&nbsp;{%- endif -%}
```

Comment:
>  I tend to think about non-breaking chars more as a typographic feature (keeping two related words together on the same line)...

Me too !

> The advantage is that a css override can be easily overriden :wink: !

Of course this is great, but you must also try to keep the link between CSS and the we are using `icheck` very obvious. Since CSS does not really have namespaces, a newcomer might not know iCheck and think this is something created within the project. This is why, in this other PR where you refactor CSS, you were kind of forced to create a `legacy.css` file containing things that might or might not still be in used, who knows?

In short, using CSS is great, but if you can avoid it without relying on hack like the `&nbsp;` hack, I would recommend avoiding it.